docs/commands/env.md

```diff
@@ -135,6 +135,7 @@ netlify env:list
 **Flags**
 
 - `context` (*string*) - Specify a deploy context or branch (contexts: "production", "deploy-preview", "branch-deploy", "dev")
+- `plain` (*boolean*) - Output environment variables as plaintext
```

Comment:
Could new entries to the help be put below `scope`? Would be nice to keep context and scope next to each other. Thanks!

Reply:
Will keep this in mind while implementing the `format` flag.

Reply:
The `plain` flag in between seemed weird to me too but the options are sorted here to produced the documentation:

https://github.com/netlify/cli/blob/d3dfa172383a21396294973a666d9665582864c4/src/utils/command-helpers.mjs#L74-L80

We can rename the option to `text` so it appear after `scope`.
Other alternatives like `raw`, `plaintext` would appear in between `context` & `scope` in the sorted order.

Let me know if you've a better name.

Reply:
Gotcha -- let's keep it as `plain`, and let the docs sort them as they currently do. But in `--help`, it would be great if context and scope could appear next to each other.

Reply:
Unfortunately, it seems like we're using the same function to order the help output too so I don't think the re-ordering would be possible unless we rename the option. The current order does appear a bit weird to me too but I don't think there's any way to override this specific instance.

Ref:

https://github.com/netlify/cli/blob/3ff4a57a95c445a185f9a210aced06bfc2b117be/src/commands/base-command.mjs#L249-L256